python/across_api/base/schema.py

```diff
@@ -0,0 +1,12 @@
+from pydantic import BaseModel, ConfigDict
+
+
+class BaseSchema(BaseModel):
+    """
+    Base class for schemas.
+
+    This class provides a base implementation for schemas and defines the `from_attributes` method.
+    Subclasses can inherit from this class and override the `from_attributes` method to define their own schema logic.
+    """
+
+    model_config = ConfigDict(from_attributes=True)
```

Comment:
I'm doing this to universally apply the ability to create Pydantic models from class attributes. 

https://docs.pydantic.dev/latest/concepts/models/#arbitrary-class-instances

This is useful because it avoids the need to covert objects into dicts when validating them against a Pydantic model. Generally it makes the code cleaner IMHO.